lib/src/util/stack_trace_mapper.dart

```diff
@@ -17,14 +18,69 @@ class StackTraceMapper {
   /// The URI of the SDK root from which dart2js loaded its sources.
   final Uri _sdkRoot;
 
-  StackTraceMapper(String contents,
+  /// The contents of the source map.
+  final String _contents;
+
+  /// The URI of the source map.
+  final Uri _mapUrl;
+
+  StackTraceMapper(this._contents,
       {Uri mapUrl, SyncPackageResolver packageResolver, Uri sdkRoot})
-      : _mapping = parseExtended(contents, mapUrl: mapUrl),
+      : _mapping = parseExtended(_contents, mapUrl: mapUrl),
```

Comment:
We don't ever actually use this in the runner isolate, right? Can we make it lazy so that we don't spend time parsing it there?